Pod/Classes/Keyboard/Keyboard.swift

```diff
@@ -33,7 +33,7 @@ import UIKit
 /**
  *  A set of abstractions around the keyboard.
  */
-final class Keyboard {
+open class Keyboard {
```

Comment:
#? Why would anyone subclass this? I think `public` would be sufficient, but it doesn't matter much.